### PR TITLE
Release/4.6.2

### PR DESCRIPTION
**Pull request by MaherKSantina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,20 +12,7 @@
 
 ## Features
 
-- OAuth2 Authentication
```
Comment (nickfrollo) [inline README.md:15]: Why is all this deleted?

File: Sources/Info.plist [hunk 2]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(SDK_VERSION)</string>
+	<string>$(MARKETING_VERSION)</string>
```
Comment (nickfrollo) [inline Sources/Info.plist:18]: Is this an Xcode change this thing?

File: README.md (hunk 1 above)
Comment (MaherKSantina) [reply README.md]: Sorry thought I need to change this, I reverted it back

File: Sources/Info.plist (hunk 2 above)
Comment (MaherKSantina) [reply Sources/Info.plist]: Not sure why this happened. Reverted back to initial version

**Comment by MaherKSantina** [comment]
Thanks @nickfrollo for the review, should I create a 4.6.2 tag and merge this to develop?

**Comment by nickfrollo** [comment]
Yep, let's merge to develop, branch to release/4.6.2 (and push) and then tag that release commit

**Pull request merged**